api/tests/test_models.py

```diff
@@ -1,6 +1,76 @@
+import pytest
+
 from api import models as api_models
+from gmn import models as gmn_models
+from gmw import models as gmw_models
+
+
+@pytest.fixture
+def organisation():
+    return api_models.Organisation(name="Nieuwegein")
 
 
-def test_organisation1():
-    organisation = api_models.Organisation(name="Nieuwegein")
+def test_organisation_name(organisation):
     assert str(organisation) == "Nieuwegein"
+
+
+def test_import_task_name(organisation):
+    import_task = api_models.ImportTask(
+        bro_domain="GMN",
+        kvk_number="123456789",
+        data_owner=organisation,
+    )
+
+    assert str(import_task) == "GMN import - Nieuwegein"
+
+
+def test_upload_task_name(organisation):
+    upload_task = api_models.UploadTask(
+        bro_domain="GMN",
+        data_owner=organisation,
+        project_number="1",
+        registration_type="GMN_StartRegistration",
+        request_type="registration",
+    )
+
+    assert str(upload_task) == "Nieuwegein: GMN_StartRegistration (registration)"
+
+
+@pytest.fixture
+def gmn(organisation):
+    return gmn_models.GMN(
+        data_owner=organisation,
+        bro_id="GMN123456789",
+    )
```

Comment:
Voor het geval er veel fixtures zijn of voor het geval je ze in andere testfiles ook wilt gebruiken: je kan ze in een `conftest.py` verzamelen, dat wordt door pytest automagisch opgepikt. 

https://docs.pytest.org/en/7.1.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files